src/modules/organisations/dto/org-members-response.dto.ts

```diff
@@ -0,0 +1,7 @@
+import UserInterface from '../../user/interfaces/UserInterface';
+
+export class OrganisationMembersResponseDto {
+  status_code: number;
+  data: Omit<Partial<UserInterface>, 'password'>[];
+  message: string;
+}
```

Comment:
These dtos should have proper decorators

Reply:
These are response dtos not input